app/server/static/js/projects.vue

```diff
@@ -1,196 +1,110 @@
+
+          div.field
+            label.label Description
+            div.control
+              textarea.textarea(v-model="description", required, placeholder="Project description")
+            p.help.is-danger {{ descriptionError }}
+
+          div.field
+            label.label Project Type
+
+            div.control
+              select(v-model="projectType", name="project_type", required)
+                option(value="", selected="selected") ---------
+                option(value="DocumentClassification") document classification
+                option(value="SequenceLabeling") sequence labeling
+                option(value="Seq2seq") sequence to sequence
+            p.help.is-danger {{ projectTypeError }}
+
+        footer.modal-card-foot.pt20.pb20.pr20.pl20.has-background-white-ter
+          button.button.is-primary(v-on:click="create()") Create
+          button.button(v-on:click="isActive = !isActive") Cancel
+
+    div.modal(v-bind:class="{ 'is-active': isDelete }")
+      div.modal-background
+      div.modal-card
+        header.modal-card-head
+          p.modal-card-title Delete Project
+          button.delete(aria-label="close", v-on:click="isDelete = !isDelete")
+        section.modal-card-body Are you sure you want to delete project?
+        footer.modal-card-foot.pt20.pb20.pr20.pl20.has-background-white-ter
+          button.button.is-danger(v-on:click="deleteProject()") Delete
+          button.button(v-on:click="isDelete = !isDelete") Cancel
+
+    section.hero
+      div.container
+        div.columns
+          div.column.is-10.is-offset-1
+            div.card.events-card
+              header.card-header
+                p.card-header-title {{ items.length }} Projects
+
+                div.field.card-header-icon
+                  div.control
+                    div.select
+                      select(v-model="selected")
+                        option(selected) All Project
+                        option Text Classification
+                        option Sequence Labeling
+                        option Seq2seq
+
+              div.card-table
+                div.content
+                  table.table.is-fullwidth
+                    tbody
+                      tr(v-for="project in selectedProjects", v-bind:key="project.id")
+                        td.pl15r
+                          div.thumbnail-wrapper.is-vertical
+                            img.project-thumbnail(
+                              v-bind:src="project.image"
+                              alt="Project thumbnail"
+                            )
+
+                          div.dataset-item__main.is-vertical
+                            div.dataset-item__main-title
+                              div.dataset-item__main-title-link.dataset-item__link
+                                a.has-text-black(v-bind:href="'/projects/' + project.id")
+                                | {{ project.name }}
```

Comment:
This line generates the following code:

```html
<div class="dataset-item__main-title-link dataset-item__link">
    <a href="/projects/2" class="has-text-black"></a>
    NER
</div>
```

The link is not clickable.

Is the following code correct?

```pug
a.has-text-black(v-bind:href="'/projects/' + project.id") {{ project.name }}
```

Reply:
Good catch. Fixed in d1fb9f4.